common/web/gesture-recognizer/src/test/auto/headless/trackedPath.js

```diff
@@ -1,70 +1,17 @@
 import { assert } from 'chai'
 import sinon  from 'sinon';
-import fs from 'fs';
-import path from 'path';
-import url from 'url';
 
 import { TrackedPath } from '@keymanapp/gesture-recognizer';
 
-import { HeadlessRecordingSimulator, timedPromise } from '../../../../build/tools/obj/index.js';
+import { timedPromise } from '../../../../build/tools/obj/index.js';
 
-// Ensures that the resources are resolved relative to this script, not to the cwd when the test
-// runner was launched.
-const scriptFolder = path.dirname(url.fileURLToPath(import.meta.url));
-const SEGMENT_TEST_JSON_FOLDER = path.resolve(`${scriptFolder}/../../resources/json/segmentation`);
-
-import { assertSegmentSimilarity } from '../../resources/assertSegmentSimilarity.js'
-;
 // End of "for the integrated style..."
 
 describe("TrackedPath", function() {
   // // File paths need to be from the package's / module's root folder
   // let testJSONtext = fs.readFileSync('src/test/resources/json/canaryRecording.json');
 
   describe("Single-sample 'sequence'", function() {
-    // A near-duplicate of the "Segmentation" -> "Single-sample 'sequence'" test.
-    // Acts as a mild 'integration' test of the segmentation engine + the public-facing
-    // TrackedPath events it advertises.
-    it("'segmentation' - expected segment types", function() {
```

Comment:
There were a couple of unit tests regarding the integration of the subsegmentation engine with the `TrackedPath` object; since they're about _integration_ with an engine no longer being integrated, they've gotta be nixed.